integrations/DCMFloodlight/utils.js

```diff
@@ -1,37 +1,61 @@
-import { isNotEmpty } from "../utils/commonUtils";
+import get from "get-value";
+import logger from "../../utils/logUtil";
+import { isDefinedAndNotNull, isNotEmpty } from "../utils/commonUtils";
 
 /**
  * transform webapp dynamicForm custom floodlight variable
  * [
       {
-        "from": "1",
-        "to": "value1"
+        "from": "property1",
+        "to": "1"
       },
       {
-        "from": "2",
-        "to": "value2"
+        "from": "property2",
+        "to": "2"
       }
   ]
- * into {u1: [value1], u2: [value2], ...}
+ * into { property1: u1, property2: u2, ... }
+ * Ref - https://support.google.com/campaignmanager/answer/2823222?hl=en
  * @param {*} customFloodlightVariable
  * @returns
  */
-function transformCustomVariable(customFloodlightVariable) {
+const transformCustomVariable = (customFloodlightVariable, message) => {
   const customVariable = {};
+  const BLACKLISTED_CHARACTERS = ['"', "<", ">", "#"];
```

Comment:
To use acceptable terms.
```suggestion
  const DENIED_CHARACTERS = ['"', "<", ">", "#"];
```